google/cloud/bigtable/data/read_rows_query.py

```diff
@@ -66,29 +84,62 @@ def __init__(
             end_key = end_key.encode()
         elif end_key is not None and not isinstance(end_key, bytes):
             raise ValueError("end_key must be a string or bytes")
+        # ensure that start_key is less than or equal to end_key
+        if start_key is not None and end_key is not None and start_key > end_key:
```

Comment:
It seems like there are 2 ways to represent infinite start/end. The bigtable native way is an empty string. We now also have None. It would good to normalize this in the constructor

Reply:
Ok, good catch. Do you mean convert empty strings to None internally?

Reply:
yep